tests/testlib/s2n_test_certs.c

```diff
@@ -26,27 +26,35 @@
 
 int s2n_read_test_pem(const char *pem_path, char *pem_out, long int max_size)
 {
-    FILE *pem_file = fopen(pem_path, "rb");
-    if (!pem_file) {
+    long int file_length = 0;
+    int ret_val = s2n_read_test_file(pem_path, pem_out, max_size, &file_length);
+    pem_out[file_length] = '\0';
```

Comment:
I'm a little curious - why move this \0 set out of the read_test_file function?

Reply:
i can revert this, I spent some time playing with manually loading .der files for tests, before I figured out it was unnecessary. This change was to avoid code duplication. In der, I didnt want the null terminator because it isn’t a c string.